ir_module_compare_wizard/README.md

```diff
@@ -0,0 +1,28 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+========================
+Ir Module Compare Wizard
+========================
+
+* Compare the modules of a xls file and the installable modules in Odoo
```

Comment:
No poner el `*`, simplemente añadir la descripción

```
Upload al list of modules with a file to check whether they exists or not and if in case
that they exist if they are also installed or not.
```